spec/models/dock_spec.rb

```diff
@@ -10,4 +10,18 @@
     it { should validate_presence_of :open }
     it { should validate_presence_of :capacity }
```

Comment:
I know you don't necessarily need it since they're all open, but it might be worthwhile to change one to closed and get practice with the boolean validations.